tools/build_help.scd

```diff
@@ -0,0 +1,7 @@
+SCDoc.helpSourceDir_("SC_HELP_SOURCE_DIR".getenv);
```

Comment:
You could also use argv instead of env variables, but I'm not which one's better? 
See [test runner](https://github.com/supercollider/supercollider/blob/develop/testsuite/scripts/sclang_test_runner.scd#L6).

Reply:
I know but I personally prefer options args instead of positional args as they make their intent more explicit - otherwise I always have to double check that I don't mix up pos0 and pos1 - and by prepending env variables to a command I still can achieve something like option args.